### ISSUE TITLE
Complex indels create improperly formatted output

### ISSUE THREAD

**Issue by gpratt** [AstraZeneca-NGS/VarDictJava#257]
Hello again,

I think I've discovered another bug, or at least some odd behavior in VarDictJava.

I think that when a read contains an insertion with an `N` _and_ there is an SNV nearby that would
normally lead to a `Complex` call then VarDictJava produces an invalid alt allele call with an `&`.

To show this I've included a minimally reproducing example with a single read.

Data set: [reproducing_data.tar.gz](https://github.com/AstraZeneca-NGS/VarDictJava/files/3602443/reproducing_data.tar.gz)

Contents:
```
tree reproducing_data/
reproducing_data/
├── example.bam
├── example.bam.bai
├── example.bed
├── example.fa
└── example.fa.fai
```

When VarDictJava is run on `example`:

```bash
/pipeline/packages/VarDictJava/build/install/VarDict/bin/VarDict \
    -N test \
    -b example.bam \
    -G example.fa \
    -p \
    -z 1 \
    -c 1 \
    -S 2 \
    -E 3 \
    -g 4 \
    -f 0.0 \
    -r 1 \
    example.bed
```

This produces the following invalid output:

```text
test	test	test	149	149	T	TTA&NT	1	1	0	0	1	0	+5/+5	1.0000	0;0	20.0	0	90.0	0	60.0	2.000	1.0000	0	0	0	0	0	1	1	AAAATAATCATCAACATAAA	CAATATATATACCATGAATA	test:149-149	Insertion	0	0
```

Notice the alt allele is `TTA&NT`. Which contains an `&`, this is invalid for most downstream tools. If I try to run anything from `picard` or anything that validates the VCF format, those tools now fail.

For completeness this issue still appears if I run the full VCF creation process.

```
/pipeline/packages/VarDictJava/build/install/VarDict/bin/VarDict \
    -N test \
    -b example.bam \
    -G example.fa \
    -p \
    -z 1 \
    -c 1 \
    -S 2 \
    -E 3 \
    -g 4 \
    -f 0.0 \
    -r 1 \
    example.bed | \
    /pipeline/packages/VarDictJava/VarDict/teststrandbias.R | \
    /pipeline/packages/VarDictJava/VarDict/var2vcf_valid.pl -N test
```

I'm not entirely sure if this is a bug in VarDictJava, or if it is a bug where the best place to fix the issue is. I'd appreciate any fixes or suggestions you might come up with!

Thank you

**Comment by gpratt** [comment]
Thanks so much for the fix advice. We'll do that for now!

**Comment by PolinaBevad** [comment]
Hi Gabriel,
We fixed this one in 1.7.0. Please try it and let me know if there will be any problems. 
Thank you!

**Comment by gpratt** [comment]
Thanks! I just tried out the new version today and it works beautifully.
